src/github/client/github-installation-client.ts

```diff
@@ -213,12 +213,12 @@ export class GitHubInstallationClient extends GitHubClient {
 		return response?.data?.data?.viewer?.repositories?.totalCount;
 	}
 
-	public async getBranchesPage(owner: string, repoName: string, perPage = 1, commitAmount = 50, cursor?: string): Promise<getBranchesResponse> {
+	public async getBranchesPage(owner: string, repoName: string, perPage = 1, timeCutoff = Date.now(), cursor?: string): Promise<getBranchesResponse> {
```

Comment:
Maybe change to `branchCommitTimeLimit`?
According to the code processing main branch, `timeCutoff` is a target Date, while here, `timeCutoff` represent miliseconds of gap between two Date.

Reply:
And maybe remove the default value? Since the calling of this function all provides default value.